src/modules/sih/sih.hpp

```diff
@@ -171,11 +182,25 @@ class Sih : public ModuleBase<Sih>, public ModuleParams, public px4::ScheduledWo
 	void publish_sih();
 	void generate_fw_aerodynamics();
 	void generate_ts_aerodynamics();
+	void sensor_step();
+
+#if defined(ENABLE_LOCKSTEP_SCHEDULER)
+	void lockstep_loop();
+	int _lockstep_component{-1};
```

Comment:
I get a compilation warning (=error in my case) because `_lockstep_component` is not used. So I commented it.